context/app/api/vitessce_confs/base_confs.py

```diff
@@ -189,9 +189,9 @@ def _check_sprm_image(self, path_regex):
         found_image_file = found_image_files[0]
         return found_image_file
 
-    def _get_ometiff_image_wrapper(self, found_image_file):
+    def _get_ometiff_image_wrapper(self, found_image_file, found_image_path):
         img_url, offsets_url = self._get_img_and_offset_url(
-            found_image_file, self._imaging_path_regex,
+            found_image_file, found_image_path,
```

Comment:
```suggestion
            found_image_file, re.escape(found_image_path),
```
If the second arg is treated as an RE, let's escape it?

Reply:
Ok!